conf/application.conf

```diff
@@ -17,4 +17,6 @@ play {
   i18n.langs = [ "en" ]
 }
 
+# max-header-size
```

Comment:
I think this might be a note that got left behind?

Reply:
yes, thanks - have removed